script.js

```diff
@@ -0,0 +1,53 @@
+var nums = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15].sort(function() {
+  return .5 - Math.random();
+});
+
+function table (x, y){  
```

Comment:
What do these arguments correspond to? (Please avoid using single-letter variable names!)